src/style.scss

```diff
@@ -78,20 +66,35 @@ img {
   }
 
   div {
+    position: relative;
+    overflow: hidden;
     width: 100%;
     height: 100%;
     margin: 0 auto;
-    background: linear-gradient(180deg, var(--auro-color-brand-midnight-200) 0%, rgba(255,255,255,1) 100%);
+    background: linear-gradient(180deg, var(--auro-color-brand-midnight-200) 0%, rgba(255 255 255 / .9) 100%);
     border-radius: 50%;
-    overflow: hidden;
-    position: relative;
   }
 
   auro-icon {
     position: absolute;
-    width: 118%;
-    left: -25%;
     top: -1%;
+    left: -25%;
+    width: 118%;
+  }
+}
+
+// maintaining boolean attr for backwards compatibility
+:host([type='sm'][tail]),
+:host([sm][tail]) {
+  display: none;
+}
+
+:host([type='md'][tail]),
+:host([md][tail]) {
+  width: var(--lg);
+
+  div {
+    width: var(--md);
   }
 }
```

Comment:
:host[lg] versions of this?

Reply:
LG is the baseline. Sm and Md are modifiers.